wemake_python_styleguide/visitors/ast/imports.py

```diff
@@ -30,6 +31,20 @@
 class _ImportsValidator(object):
     """Utility class to separate logic from the visitor."""
 
+    _vague_imports_blacklist = {
```

Comment:
We use `frozenset` and `ClassVar` type

Reply:
`frozenset` makes sense, but how does `ClassVar` play into this?

Reply:
That's how we separate class-level variables from instance-level variables: `_vague_imports_blacklist: ClassVar[FrozenSet[str]]`. But, it does not really matter, because it should be moved to `constants.py` as described in one of my comments.